.cirrus.yml

```diff
@@ -524,8 +524,13 @@ compose_test_task:
 windows_smoke_test_task:
     name: "Windows Smoke Test"
     alias: windows_smoke_test
-    # Don't run for multiarch/container image cirrus-cron job.
-    only_if: $CIRRUS_CRON != 'multiarch'
+    # Only run for non-docs/copr PRs and non-multiarch branch builds
+    # and never for tags.  Docs: ./contrib/cirrus/CIModes.md
+    only_if: >-
+        $CIRRUS_TAG == '' &&
+        $CIRRUS_CRON != 'multiarch' &&
+        $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' &&
+        $CIRRUS_CHANGE_TITLE !=~ '.*CI:COPR.*' &&
```

Comment:
i guess the last `&&` needs to be removed?

Reply:
Oh woops, copy-paste error.  Thanks.